com/abweichung.go

```diff
@@ -1,9 +1,24 @@
 package com
 
-import "github.com/hochfrequenz/go-bo4e/enum/abweichungsgrund"
+import (
+	"github.com/go-playground/validator/v10"
+	"github.com/hochfrequenz/go-bo4e/enum/abweichungsgrund"
+)
 
 type Abweichung struct {
 	Referenz                  string                             `json:"referenz,omitempty"`
 	AbweichungsGrund          *abweichungsgrund.AbweichungsGrund `json:"abweichungsgrund,omitempty" validate:"omitempty"`
-	AbweichungsGrundBemerkung string                             `json:"abweichungsgrundBemerkung,omitempty"`
+	AbweichungsGrundBemerkung *string                            `json:"abweichungsgrundBemerkung,omitempty"`
+	AbweichungsgrundCode      *string                            `json:"abweichungsgrundCode,omitempty" validate:"omitempty"`
+	AbweichungsgrundCodeliste *string                            `json:"abweichungsgrundCodeliste,omitempty" validate:"omitempty"`
```

Comment:
```suggestion
	AbweichungsgrundCodeliste *string                            `json:"abweichungsgrundCodeliste,omitempty" validate:"required_with=AbweichungsgrundCode,omitempty"`
```
das führt dazu/soll dazu führen (ich hab es nicht getestet), dass die codeliste anzugeben ist, wenn ein code angegeben wurde.

https://github.com/go-playground/validator/blob/dd2857a4cb6c53af5bf6944cc15fd04c865d28ae/doc.go#L292

Reply:
die Doku hatte ich nicht gefunden, hat aber irgendwie im unit test nicht funktioniert; ich gehe jetzt erstmal mit dem stand ins Rennen